testing_server.py

```diff
@@ -0,0 +1,105 @@
+# import sys
```

Comment:
Is this file still needed? If not, delete it.

Reply:
you can delete it ...